src/commands/new/contract.rs

```diff
@@ -75,4 +76,15 @@ mod tests {
 		}
 		Ok(())
 	}
+
+	#[test]
+	fn test_new_contract_command_execute_fails_path_no_exist() -> Result<(), Error> {
+		let mut path = PathBuf::new();
+		path.push("test_contract");
+		let command =
+			NewContractCommand { name: "test_contract".to_string(), path: Some(path) };
+		let result_error = command.execute();
```

Comment:
https://github.com/r0gue-io/pop-cli/actions/runs/8411223051/job/23030577025?pr=83#step:11:1335
- Looks like the CI is failing here.